vignettes/UsingSelfControlledCohort.Rmd

```diff
@@ -0,0 +1,222 @@
+  participant Tn30 as Unexposed Risk Window Start
+  participant Tn1 as Unexposed Risk Window End
+  participant T0 as Exposure Start Date (Time 0)
+  participant T1 as Exposed Risk Window Start
+  participant T30 as Exposed Risk Window End
+  participant T100 as Observation Period End
+  
+
+  Tn100 ->> T0: washout period
+  T0 ->> T100: followup period
+  Tn30->>Tn1: Unexposed time at risk
+  T1->>T30: Exposed time at risk
+")
+```
+
+
+Only people who experience the exposure are used in the analysis and their exposure start date is taken as the index date or *Time 0*. The unexposed risk window is defined prior to Time 0 and represents a duration of time when the person was not exposed but could have experienced the outcome. The exposed risk window is defined as a period of time after the Time 0 and represents a duration of time when the person was exposed and could have experienced the outcome. Both of these time windows must be contained within a continuous period of observation. The details about when the risk windows start and end can be adjusted to meet the analysis specifications.
+
+The time between the start of the observation period and Time 0 is called the washout period and a minimum allowed washout period can be set when defining the analysis. Similarly the time between Time 0 and the observation period end date is the followup period. A minimum followup period can also be set.
+
+A common type of exposure is exposure to a drug but an exposure could be any set of characteristics that are captured in your data. We will look at a few different ways to define exposures.
+
+# Tables used in the analysis
+
+Conceptually there are two primary tables used in the analysis along with a few supporting tables that are part of the OMOP Common Data Model
+
+```{r, echo=F}
+exposure_table <- data.frame(field    = c("person_id", "exposure_concept_id", "exposure_start_date", "exposure_end_date"),
+                             datatype = c("INTEGER",    "INTEGER",             "DATE",               "DATE"))
+
+outcome_table <- data.frame(field    = c("person_id", "outcome_concept_id", "outcome_start_date"),
+                            datatype = c("INTEGER",   "INTEGER",            "DATE"))
+
+knitr::kable(exposure_table)
+knitr::kable(outcome_table)
+```
+
+
+These tables can be created manually or standard CDM tables can be used (eg. drug_era => exposure_table, condition_era => outcome_table). Every exposure-outcome combination in these tables will be included in the analysis unless otherwise specified. The additional CDM tables that are required for this analyis are `observation_period` (for observation start and end dates) and `person` (for year of birth).
+
+# A first example
+
+The SelfControlledCohort package makes it easy to run examine the association between every drug and every condition in a CDM database. The Eunomia package provides a mini CDM that runs entirely within R that we can use for examples.
```

Comment:
Can you clarify this sentance, is the word "run" a typo?

Reply:
Yes that’s a typo.

Reply:
Fixed.